MediaGalleryIntegration/Test/Unit/Model/OpenDialogUrlProviderTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryIntegration\Test\Unit\Model;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use Magento\MediaGalleryIntegration\Model\OpenDialogUrlProvider;
+use Magento\MediaGalleryUiApi\Api\ConfigInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit test for \Magento\MediaGalleryIntegration\Plugin\UpdateWysiwygDialogUrlTinyMceTest
+ */
+class OpenDialogUrlProviderTest extends TestCase
```

Comment:
I would be good to cover `Magento\MediaGalleryIntegration\Model\OpenDialogUrlProvider` by integration test instead of the unit test:
1. Use `@magentoConfigFixture default/system/media_gallery/enabled` fixture to set the system state
2. Call `getUrl` method